osu.Server.Spectator/Properties/launchSettings.json

```diff
@@ -14,6 +14,7 @@
       "launchBrowser": false,
       "launchUrl": "ranklookup",
```

Comment:
pretty sure this configuration is completely invalid (was copied from another project). might as well remove it.

Reply:
you mean removing the whole IIS express profile?

Reply:
correct.